app/assets/javascripts/short-form/components/lendingInstitution.js.coffee

```diff
@@ -28,12 +29,19 @@ angular.module('dahlia.components')
         else
           false
 
+      @agentIsInactive = (id) ->
+        return unless id
+        selectedAgent = _.find(@agents, { Id: id })
+        selectedAgent.Status == 'Inactive'
+
       @lendingInstitutionsNames = ->
         _.keys(@lendingInstitutions)
 
       @onChangeLendingInstitution = ->
         @application.lendingAgent = null
-        @agents = @lendingInstitutions[@selectedInstitution]
+        @agents = _.filter(@lendingInstitutions[@selectedInstitution], (agent) ->
+          agent.Status != 'Removed'
```

Comment:
this shouldn't be necessary. No lending agents with status removed should be returned from salesforce, so when building the available lending institutions if the lending institution doesn't have any active or inactive officers (only removed) then it won't be in the list.